geode-gfsh/src/integrationTest/java/org/apache/geode/management/internal/cli/GfshParserConverterTest.java

```diff
@@ -103,7 +103,7 @@ public void testJsonKey() {
   public void testUnspecifiedValueToStringArray() {
     String command = "change loglevel --loglevel=finer --groups=group1,group2";
     ParseResult result = parser.parse(command);
-    String[] memberIdValue = (String[]) result.getArguments()[0];
```

Comment:
Why did this switch to 1?

Reply:
Due to the change in the order of parameters in `ChangeLogLevelCommand`, the memberId is returned in second position by the parser, so I had to modify the array index from 0 to 1.